src/wallet/wallet.h

```diff
@@ -64,6 +64,8 @@ static const int DEFAULT_CUSTOMBACKUPTHRESHOLD = 1;
 static const bool DEFAULT_AUTOCONVERTADDRESS = true;
 //! if set, all keys will be derived by using BIP32
 static const bool DEFAULT_USE_HD_WALLET = true;
+//! if set,will show warning if the wallet is a hd wallet and is unencrypted
+static const bool DEFAULT_ENABLE_WARN_ENCRYPTHD = false;
```

Comment:
This doesn't seem like a good idea; to render that warning disabled.  It might be worth looking at the tests and fix the test to allow for it; rather than disabling this warning.

Reply:
all tests fail when it is enabled,for some reason it counts the warning as a error in the tests,hence why i added it

Reply:
will try setting it to true now that all tests passes